examples/flax_mnist.py

```diff
@@ -15,16 +15,32 @@
 """MNIST example with Flax and JAXopt."""
 
 from absl import app
+from absl import flags
+
 from flax import linen as nn
+
 import jax
 import jax.numpy as jnp
+
 from jaxopt import loss
 from jaxopt import OptaxSolver
+from jaxopt import PolyakSGD
 from jaxopt import tree_util
+
 import optax
+
 import tensorflow_datasets as tfds
 
 
+flags.DEFINE_float("l2_regul", 1e-4, "L2 regularization.")
```

Comment:
is this the right way to set the L2 regularization?. Other examples call it lam (ridge_reg_implicit_diff.py)  and l2reg in dataset_distillation.py